server/model/feature_flags.go

```diff
@@ -80,6 +80,8 @@ type FeatureFlags struct {
 	AppsSidebarCategory bool
 
 	CloudReverseTrial bool
+
+	NewUI bool
```

Comment:
should we be more descriptive here. It might be misleading to as new UI of whole app or certain components.

Reply:
CompassComponentsUI ? or anything

Reply:
NewUI sounds better to me because we will using the same for some Modal revamps as well.(Not Migration)